bitnami/mxnet/templates/deployment-pvc.yaml

```diff
@@ -22,5 +22,7 @@ spec:
   {{- else }}
   storageClassName: "{{ .Values.persistence.storageClass }}"
   {{- end }}
+  {{- else if .Values.global.storageClass }}
```

Comment:
Move this logic to set the `storageClassName` to a macro function in the `_helpers.tpl`

Reply:
The `global.storageClass` value should have more priority than `persistence.storageClass` one. Check how `global.imagePullSecrets` is implemented.